templates/address-space-controller/020-ConfigMap-standard-broker-definitions.yaml

```diff
@@ -115,15 +109,26 @@ data:
               - name: AUTHENTICATION_SERVICE_SASL_INIT_HOST
                 value: ${AUTHENTICATION_SERVICE_SASL_INIT_HOST}
               - name: AUTHENTICATION_SERVICE_OAUTH_URL
-                value: ${AUTHENTICATION_SERVICE_OAUTH_URL}
+                value: ${AUTHENTICATION_SERVICE_OAUTH_URL}                
+              - name: HOME
+                value: ${HOME}
+              - name: AMQ_NAME
+                value: ${AMQ_NAME}                
               image: ${ARTEMIS_IMAGE}
               imagePullPolicy: ${IMAGE_PULL_POLICY}
+              command:
+              - /opt/apache-artemis/custom/bin/launch-broker.sh
+              lifecycle:
+                preStop:
+                  exec:
+                    command:
+                    - $ARTEMIS_HOME/custom/shutdown-hook/shutdown-hook.sh
```

Comment:
Remove the lifecycle hook as it has been removed from master.

Reply:
now removed